library/scrollphat/IS31FL3730.py

```diff
@@ -1,22 +1,25 @@
 MODE_5X11 = 0b00000011
 
-class IS31FL3730:
-    def __init__(self, smbus, font):
-        self.smbus = smbus
-        self.font = font
-
+class I2cConstants:
+    def __init__(self):
         self.I2C_ADDR = 0x60
         self.CMD_SET_MODE = 0x00
         self.CMD_SET_BRIGHTNESS = 0x19
         self.MODE_5X11 = 0b00000011
 
+class IS31FL3730:
+    def __init__(self, smbus, font):
+        self.bus = smbus
+        self.font = font
+        self.i2cConstants = I2cConstants()
+
     def initialize(self):
         self.bus = self.smbus.SMBus(1)
```

Comment:
@alexellis I fixed what I believe is a typo/bug on this line in https://github.com/pimoroni/scroll-phat/commit/7c7adc2d2b23ae7472135ecb023b5443feb09ec8, can you confirm I've got the right end of the stick?